src/locales/cy/translation.json

```diff
@@ -72,6 +72,10 @@
         "pageTitle": "Hysbysiad preifatrwydd",
         "linkText": "Hysbysiad preifatrwydd"
       },
+      "support": {
+        "pageTitle": "Cefnogaeth",
```

Comment:
Where is `pageTitle` used?

Reply:
Hi @danacotoran , good catch , page title is not needed as part of this ticket, removing.